File: assets/samples/email.verification/php/email_api.php
<?php
$email = 'support@whoisxmlapi.com';
$api_key = 'your_api_key';
$api_url = 'https://emailverification.whoisxmlapi.com/api/v1';

$url = "{$api_url}?apiKey={$api_key}&emailAddress={$email}";

print(file_get_contents($url));

File: assets/samples/email.verification/python/email_api.py
try:
    from urllib.request import urlopen
except ImportError:
    from urllib2 import urlopen

email = 'support@whoisxmlapi.com'
api_key = 'your_api_key'
api_url = 'https://emailverification.whoisxmlapi.com/api/v1?'

url = api_url + 'apiKey=' + api_key + '&emailAddress=' + email

print(urlopen(url).read().decode('utf8'))
